readme.md

```diff
@@ -171,30 +177,31 @@ Once installation is working, try creating and requesting resources. Note that t
 * **Question:** You may notice that you can add custom error messages to individual fields failing. Try adding these and take a look at the error message received. How can you make use of those specific messages for each field?
 
 * **Your Answer:**
-
+By adding a constraint failure message to the schema definition: required:[true, 'a start year is required']
```

Comment:
You can get access to these messages by using a `try/catch` or a `.catch()` and digging into the provided error.